diff --git a/src/Controller/SongController.php b/src/Controller/SongController.php
index c13619f..ef15be1 100644
--- a/src/Controller/SongController.php
+++ b/src/Controller/SongController.php
@@ -3,6 +3,7 @@
 namespace App\Controller;
 
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
@@ -11,6 +12,13 @@ class SongController extends AbstractController
     #[Route('/api/songs/{id}')]
     public function getSong($id): Response
     {
-        dd($id);
+        // TODO query the database
+        $song = [
+            'id' => $id,
+            'name' => 'Waterfalls',
+            'url' => 'https://symfonycasts.s3.amazonaws.com/sample.mp3',
+        ];
+
+        return new JsonResponse($song);
     }
 }
