diff --git a/src/Controller/SongController.php b/src/Controller/SongController.php
index 07c1dcc..c140ac9 100644
--- a/src/Controller/SongController.php
+++ b/src/Controller/SongController.php
@@ -9,7 +9,7 @@ use Symfony\Component\Routing\Annotation\Route;
 
 class SongController extends AbstractController
 {
-    #[Route('/api/songs/{id}')]
+    #[Route('/api/songs/{id}', methods: ['GET'])]
     public function getSong($id): Response
     {
         // TODO query the database
