diff --git a/src/Controller/SongController.php b/src/Controller/SongController.php
index ef15be1..07c1dcc 100644
--- a/src/Controller/SongController.php
+++ b/src/Controller/SongController.php
@@ -19,6 +19,6 @@ class SongController extends AbstractController
             'url' => 'https://symfonycasts.s3.amazonaws.com/sample.mp3',
         ];
 
-        return new JsonResponse($song);
+        return $this->json($song);
     }
 }
