diff --git a/src/Controller/SongController.php b/src/Controller/SongController.php
index 9d4fce4..59bfc14 100644
--- a/src/Controller/SongController.php
+++ b/src/Controller/SongController.php
@@ -9,7 +9,7 @@ use Symfony\Component\Routing\Annotation\Route;
 
 class SongController extends AbstractController
 {
-    #[Route('/api/songs/{id}', methods: ['GET'])]
+    #[Route('/api/songs/{id<\d+>}', methods: ['GET'])]
     public function getSong(int $id): Response
     {
         // TODO query the database
