diff --git a/src/Controller/SongController.php b/src/Controller/SongController.php
index 27274c4..1d62550 100644
--- a/src/Controller/SongController.php
+++ b/src/Controller/SongController.php
@@ -18,6 +18,6 @@ class SongController extends AbstractController
             'url' => 'https://symfonycasts.s3.amazonaws.com/sample.mp3',
         ];
 
-        return new JsonResponse($song);
+        return $this->json($song);
     }
 }
