diff --git a/src/Controller/SongController.php b/src/Controller/SongController.php
index 1d62550..0fbc735 100644
--- a/src/Controller/SongController.php
+++ b/src/Controller/SongController.php
@@ -8,7 +8,7 @@ use Symfony\Component\Routing\Annotation\Route;
 
 class SongController extends AbstractController
 {
-    #[Route('/api/songs/{id}')]
+    #[Route('/api/songs/{id}', methods: ['GET'])]
     public function getSong($id)
     {
         // TODO query the database
