diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index 2e844f8..95d1465 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -40,29 +40,4 @@ class VinylController extends AbstractController
             'mixes' => $mixes,
         ]);
     }
-
-    private function getMixes(): array
-    {
-        // temporary fake "mixes" data
-        return [
-            [
-                'title' => 'PB & Jams',
-                'trackCount' => 14,
-                'genre' => 'Rock',
-                'createdAt' => new \DateTime('2021-10-02'),
-            ],
-            [
-                'title' => 'Put a Hex on your Ex',
-                'trackCount' => 8,
-                'genre' => 'Heavy Metal',
-                'createdAt' => new \DateTime('2022-04-28'),
-            ],
-            [
-                'title' => 'Spice Grills - Summer Tunes',
-                'trackCount' => 10,
-                'genre' => 'Pop',
-                'createdAt' => new \DateTime('2019-06-20'),
-            ],
-        ];
-    }
 }
