diff --git a/templates/base.html.twig b/templates/base.html.twig
index cca65a2..a9dd883 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -13,7 +13,7 @@
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@300;400;700&display=swap" rel="stylesheet">
-        <link rel="stylesheet" href="/styles/app.css">
+        <link rel="stylesheet" href="{{ asset('styles/app.css') }}">
 
         {# Run `composer require symfony/webpack-encore-bundle` to start using Symfony UX #}
         {% block stylesheets %}
