diff --git a/assets/bootstrap.js b/assets/bootstrap.js
index 4ab2df6..8e05737 100644
--- a/assets/bootstrap.js
+++ b/assets/bootstrap.js
@@ -1,11 +1,7 @@
-import { startStimulusApp } from '@symfony/stimulus-bridge';
+import { startStimulusApp } from '@symfony/stimulus-bundle';
 
 // Registers Stimulus controllers from controllers.json and in the controllers/ directory
-export const app = startStimulusApp(require.context(
-    '@symfony/stimulus-bridge/lazy-controller-loader!./controllers',
-    true,
-    /\.[jt]sx?$/
-));
+export const app = startStimulusApp();
 
 // register any custom, 3rd party controllers here
 // app.register('some_controller_name', SomeImportedController);
