diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index 30d6e5d..0839cad 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -10,6 +10,12 @@ use function Symfony\Component\String\u;
 
 class VinylController extends AbstractController
 {
+    public function __construct(
+        private bool $isDebug,
+        private MixRepository $mixRepository
+    )
+    {}
+
     #[Route('/', name: 'app_homepage')]
     public function homepage(): Response
     {
@@ -29,12 +35,12 @@ class VinylController extends AbstractController
     }
 
     #[Route('/browse/{slug}', name: 'app_browse')]
-    public function browse(MixRepository $mixRepository, bool $isDebug, string $slug = null): Response
+    public function browse(string $slug = null): Response
     {
-        dump($isDebug);
+        dump($this->isDebug);
         $genre = $slug ? u(str_replace('-', ' ', $slug))->title(true) : null;
 
-        $mixes = $mixRepository->findAll();
+        $mixes = $this->mixRepository->findAll();
 
         return $this->render('vinyl/browse.html.twig', [
             'genre' => $genre,
