diff --git a/config/services.yaml b/config/services.yaml
index 11bead9..c6d06ad 100644
--- a/config/services.yaml
+++ b/config/services.yaml
@@ -15,8 +15,8 @@ services:
     _defaults:
         autowire: true      # Automatically injects dependencies in your services.
         autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
-#        bind:
-#            'bool $isDebug': '%kernel.debug%'
+        bind:
+            'bool $isDebug': '%kernel.debug%'
 
     # makes classes in src/ available to be used as services
     # this creates a service per class whose id is the fully-qualified class name
diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index f014c3c..30d6e5d 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -29,8 +29,9 @@ class VinylController extends AbstractController
     }
 
     #[Route('/browse/{slug}', name: 'app_browse')]
-    public function browse(MixRepository $mixRepository, string $slug = null): Response
+    public function browse(MixRepository $mixRepository, bool $isDebug, string $slug = null): Response
     {
+        dump($isDebug);
         $genre = $slug ? u(str_replace('-', ' ', $slug))->title(true) : null;
 
         $mixes = $mixRepository->findAll();
