diff --git a/src/Entity/VinylMix.php b/src/Entity/VinylMix.php
index 2a3050b..bc4945d 100644
--- a/src/Entity/VinylMix.php
+++ b/src/Entity/VinylMix.php
@@ -113,4 +113,11 @@ class VinylMix
 
         return $this;
     }
+
+    public function getVotesString(): string
+    {
+        $prefix = ($this->votes === 0) ? '' : (($this->votes >= 0) ? '+' : '-');
+
+        return sprintf('%s %d', $prefix, abs($this->votes));
+    }
 }
diff --git a/templates/vinyl/browse.html.twig b/templates/vinyl/browse.html.twig
index 61fa392..20cfbbd 100644
--- a/templates/vinyl/browse.html.twig
+++ b/templates/vinyl/browse.html.twig
@@ -37,7 +37,7 @@
                     |
                     <span>{{ mix.createdAt|ago }}</span>
                     <br>
-                    {{ mix.votes }} votes
+                    {{ mix.votesString }} votes
                 </div>
             </div>
             {% endfor %}
