diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index 9a3d0ee..e72d943 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -35,7 +35,6 @@ class VinylController extends AbstractController
     {
         $genre = $slug ? u(str_replace('-', ' ', $slug))->title(true) : null;
 
-        dump($cache);
         $mixes = $cache->get('mixes_data', function(CacheItemInterface $cacheItem) use ($httpClient) {
             $cacheItem->expiresAfter(5);
             $response = $httpClient->request('GET', 'https://raw.githubusercontent.com/SymfonyCasts/vinyl-mixes/main/mixes.json');
