diff --git a/templates/vinyl/browse.html.twig b/templates/vinyl/browse.html.twig
index b1a2cbf..4767f77 100644
--- a/templates/vinyl/browse.html.twig
+++ b/templates/vinyl/browse.html.twig
@@ -25,7 +25,7 @@
 
     <div>
         <h2 class="mt-5">Mixes</h2>
-        <turbo-frame id="mix-browse-list">
+        <turbo-frame id="mix-browse-list-{{ pager.currentPage }}">
             <div class="row">
                 {% for mix in pager %}
                 <div class="col col-md-4">
