diff --git a/src/Factory/VinylMixFactory.php b/src/Factory/VinylMixFactory.php
index 0a4ac8e..d211689 100644
--- a/src/Factory/VinylMixFactory.php
+++ b/src/Factory/VinylMixFactory.php
@@ -39,10 +39,10 @@ final class VinylMixFactory extends ModelFactory
     {
         return [
             'title' => self::faker()->words(5, true),
+            'description' => self::faker()->paragraph(),
             'trackCount' => self::faker()->numberBetween(5, 20),
             'genre' => self::faker()->randomElement(['pop', 'rock']),
             'votes' => self::faker()->numberBetween(-50, 50),
-            'slug' => self::faker()->text(),
         ];
     }
 
