diff --git a/templates/vinyl/browse.html.twig b/templates/vinyl/browse.html.twig
index 160992d..0b64729 100644
--- a/templates/vinyl/browse.html.twig
+++ b/templates/vinyl/browse.html.twig
@@ -7,13 +7,19 @@
     <h2 class="mt-5 mb-4">Filter by Genre</h2>
     <ul class="genre-list ps-0 mt-2 mb-3">
         <li class="d-inline">
-            <a class="btn btn-primary btn-sm" href="#">Pop</a>
+            <a class="btn btn-primary btn-sm" href="{{ path('app_browse', {
+                slug: 'pop'
+            }) }}">Pop</a>
         </li>
         <li class="d-inline">
-            <a class="btn btn-primary btn-sm" href="#">Rock</a>
+            <a class="btn btn-primary btn-sm" href="{{ path('app_browse', {
+                slug: 'rock'
+            }) }}">Rock</a>
         </li>
         <li class="d-inline">
-            <a class="btn btn-primary btn-sm" href="#">Heavy Metal</a>
+            <a class="btn btn-primary btn-sm" href="{{ path('app_browse', {
+                slug: 'all-accordion'
+            }) }}">All Accordion</a>
         </li>
     </ul>
 
