diff --git a/src/Command/TalkToMeCommand.php b/src/Command/TalkToMeCommand.php
index 568eff3..71f62dd 100644
--- a/src/Command/TalkToMeCommand.php
+++ b/src/Command/TalkToMeCommand.php
@@ -12,15 +12,15 @@ use Symfony\Component\Console\Style\SymfonyStyle;
 
 #[AsCommand(
     name: 'app:talk-to-me',
-    description: 'Add a short description for your command',
+    description: 'A self-aware command that can do... only one thing.',
 )]
 class TalkToMeCommand extends Command
 {
     protected function configure(): void
     {
         $this
-            ->addArgument('arg1', InputArgument::OPTIONAL, 'Argument description')
-            ->addOption('option1', null, InputOption::VALUE_NONE, 'Option description')
+            ->addArgument('name', InputArgument::OPTIONAL, 'Your name')
+            ->addOption('yell', null, InputOption::VALUE_NONE, 'Shall I yell?')
         ;
     }
 
