diff --git a/src/Entity/VinylMix.php b/src/Entity/VinylMix.php
index b4f8006..ec1954d 100644
--- a/src/Entity/VinylMix.php
+++ b/src/Entity/VinylMix.php
@@ -29,6 +29,9 @@ class VinylMix
     #[ORM\Column]
     private ?\DateTimeImmutable $createdAt = null;
 
+    #[ORM\Column]
+    private ?int $votes = null;
+
     public function getId(): ?int
     {
         return $this->id;
@@ -93,4 +96,16 @@ class VinylMix
 
         return $this;
     }
+
+    public function getVotes(): ?int
+    {
+        return $this->votes;
+    }
+
+    public function setVotes(int $votes): self
+    {
+        $this->votes = $votes;
+
+        return $this;
+    }
 }
