diff --git a/src/Entity/VinylMix.php b/src/Entity/VinylMix.php
index 833b847..244c65c 100644
--- a/src/Entity/VinylMix.php
+++ b/src/Entity/VinylMix.php
@@ -28,6 +28,9 @@ class VinylMix
     #[ORM\Column(type: 'datetime_immutable')]
     private $createdAt;
 
+    #[ORM\Column(type: 'integer')]
+    private $votes;
+
     public function getId(): ?int
     {
         return $this->id;
@@ -92,4 +95,16 @@ class VinylMix
 
         return $this;
     }
+
+    public function getVotes(): ?int
+    {
+        return $this->votes;
+    }
+
+    public function setVotes(int $votes): self
+    {
+        $this->votes = $votes;
+
+        return $this;
+    }
 }
