diff --git a/src/Service/MixRepository.php b/src/Service/MixRepository.php
index bd58e82..fb71bfe 100644
--- a/src/Service/MixRepository.php
+++ b/src/Service/MixRepository.php
@@ -10,7 +10,7 @@ use Symfony\Contracts\HttpClient\HttpClientInterface;
 class MixRepository
 {
     public function __construct(
-        private HttpClientInterface $httpClient,
+        private HttpClientInterface $githubContentClient,
         private CacheInterface $cache,
         #[Autowire('%kernel.debug%')]
         private bool $isDebug
@@ -20,7 +20,7 @@ class MixRepository
     {
         return $this->cache->get('mixes_data', function(CacheItemInterface $cacheItem) {
             $cacheItem->expiresAfter($this->isDebug ? 5 : 60);
-            $response = $this->httpClient->request('GET', '/SymfonyCasts/vinyl-mixes/main/mixes.json');
+            $response = $this->githubContentClient->request('GET', '/SymfonyCasts/vinyl-mixes/main/mixes.json');
 
             return $response->toArray();
         });
