diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index f7aab52..6e5a397 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -2,7 +2,6 @@
 
 namespace App\Controller;
 
-use Knp\Bundle\TimeBundle\DateTimeFormatter;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
@@ -29,15 +28,11 @@ class VinylController extends AbstractController
     }
 
     #[Route('/browse/{slug}', name: 'app_browse')]
-    public function browse(DateTimeFormatter $timeFormatter, string $slug = null): Response
+    public function browse(string $slug = null): Response
     {
         $genre = $slug ? u(str_replace('-', ' ', $slug))->title(true) : null;
         $mixes = $this->getMixes();
 
-        foreach ($mixes as $key => $mix) {
-            $mixes[$key]['ago'] = $timeFormatter->formatDiff($mix['createdAt']);
-        }
-
         return $this->render('vinyl/browse.html.twig', [
             'genre' => $genre,
             'mixes' => $mixes,
diff --git a/templates/vinyl/browse.html.twig b/templates/vinyl/browse.html.twig
index 66b9faa..4963013 100644
--- a/templates/vinyl/browse.html.twig
+++ b/templates/vinyl/browse.html.twig
@@ -35,7 +35,7 @@
                     |
                     <span>{{ mix.genre }}</span>
                     |
-                    <span>{{ mix.ago }}</span>
+                    <span>{{ mix.createdAt|ago }}</span>
                 </div>
             </div>
             {% endfor %}
