diff --git a/src/Service/MixRepository.php b/src/Service/MixRepository.php
index 7b90f7b..4a8e639 100644
--- a/src/Service/MixRepository.php
+++ b/src/Service/MixRepository.php
@@ -10,13 +10,14 @@ class MixRepository
 {
     public function __construct(
         private HttpClientInterface $httpClient,
-        private CacheInterface $cache
+        private CacheInterface $cache,
+        private bool $isDebug
     ) {}
 
     public function findAll(): array
     {
         return $this->cache->get('mixes_data', function(CacheItemInterface $cacheItem) {
-            $cacheItem->expiresAfter(5);
+            $cacheItem->expiresAfter($this->isDebug ? 5 : 60);
             $response = $this->httpClient->request('GET', 'https://raw.githubusercontent.com/SymfonyCasts/vinyl-mixes/main/mixes.json');
 
             return $response->toArray();
