diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index 6ea8466..ecb582a 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -35,7 +35,7 @@ class VinylController extends AbstractController
     {
         $genre = $slug ? u(str_replace('-', ' ', $slug))->title(true) : null;
 
-        $mixes = $mixRepository->findAll();
+        $mixes = $mixRepository->findAll($httpClient, $cache);
 
         return $this->render('vinyl/browse.html.twig', [
             'genre' => $genre,
