diff --git a/templates/vinyl/browse.html.twig b/templates/vinyl/browse.html.twig
index 62cd7e3..8531dfb 100644
--- a/templates/vinyl/browse.html.twig
+++ b/templates/vinyl/browse.html.twig
@@ -26,7 +26,7 @@
     <div>
         <h2 class="mt-5">Mixes</h2>
         <div class="row">
-            {% for mix in mixes %}
+            {% for mix in pager %}
             <div class="col col-md-4">
                 <a href="{{ path('app_mix_show', {
                     slug: mix.slug
