diff --git a/src/Controller/MixController.php b/src/Controller/MixController.php
index e9b0334..8ab5c20 100644
--- a/src/Controller/MixController.php
+++ b/src/Controller/MixController.php
@@ -33,14 +33,8 @@ class MixController extends AbstractController
     }
 
     #[Route('/mix/{id}', name: 'app_mix_show')]
-    public function show($id, VinylMixRepository $mixRepository): Response
+    public function show(VinylMix $mix): Response
     {
-        $mix = $mixRepository->find($id);
-
-        if (!$mix) {
-            throw $this->createNotFoundException('Mix not found');
-        }
-
         return $this->render('mix/show.html.twig', [
             'mix' => $mix,
         ]);
