diff --git a/templates/vinyl/browse.html.twig b/templates/vinyl/browse.html.twig
index 4963013..b0ba2c7 100644
--- a/templates/vinyl/browse.html.twig
+++ b/templates/vinyl/browse.html.twig
@@ -29,7 +29,7 @@
             {% for mix in mixes %}
             <div class="col col-md-4">
                 <div class="mixed-vinyl-container p-3 text-center">
-                    <img src="https://via.placeholder.com/300" data-src="https://via.placeholder.com/300" alt="Square placeholder img">
+                    <img src="https://via.placeholder.com/300" alt="Square placeholder img">
                     <p class="mt-2"><strong>{{ mix.title }}</strong></p>
                     <span>{{ mix.trackCount }} Tracks</span>
                     |
