diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index 97a69bf..04a4634 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -20,7 +20,6 @@ class VinylController extends AbstractController
             ['song' => 'On Bended Knee', 'artist' => 'Boyz II Men'],
             ['song' => 'Fantasy', 'artist' => 'Mariah Carey'],
         ];
-        dump($tracks);
 
         return $this->render('vinyl/homepage.html.twig', [
             'title' => 'PB & Jams',
diff --git a/templates/vinyl/homepage.html.twig b/templates/vinyl/homepage.html.twig
index b622baf..2fc8535 100644
--- a/templates/vinyl/homepage.html.twig
+++ b/templates/vinyl/homepage.html.twig
@@ -10,6 +10,7 @@
 <div>
     Tracks:
 
+    {{ dump(tracks) }}
     <ul>
         {% for track in tracks %}
             <li>
