diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index 04a4634..dba8c43 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -20,6 +20,7 @@ class VinylController extends AbstractController
             ['song' => 'On Bended Knee', 'artist' => 'Boyz II Men'],
             ['song' => 'Fantasy', 'artist' => 'Mariah Carey'],
         ];
+        dd($tracks);
 
         return $this->render('vinyl/homepage.html.twig', [
             'title' => 'PB & Jams',
