diff --git a/templates/vinyl/browse.html.twig b/templates/vinyl/browse.html.twig
index b0ba2c7..61fa392 100644
--- a/templates/vinyl/browse.html.twig
+++ b/templates/vinyl/browse.html.twig
@@ -36,6 +36,8 @@
                     <span>{{ mix.genre }}</span>
                     |
                     <span>{{ mix.createdAt|ago }}</span>
+                    <br>
+                    {{ mix.votes }} votes
                 </div>
             </div>
             {% endfor %}
