diff --git a/src/Service/MixRepository.php b/src/Service/MixRepository.php
deleted file mode 100644
index 87ee9d6..0000000
--- a/src/Service/MixRepository.php
+++ /dev/null
@@ -1,40 +0,0 @@
-<?php
-
-namespace App\Service;
-
-use Psr\Cache\CacheItemInterface;
-use Symfony\Bridge\Twig\Command\DebugCommand;
-use Symfony\Component\Console\Input\ArrayInput;
-use Symfony\Component\Console\Output\BufferedOutput;
-use Symfony\Contracts\Cache\CacheInterface;
-use Symfony\Component\DependencyInjection\Attribute\Autowire;
-use Symfony\Contracts\HttpClient\HttpClientInterface;
-use Twig\Loader\FilesystemLoader;
-
-class MixRepository
-{
-    public function __construct(
-        private HttpClientInterface $githubContentClient,
-        private CacheInterface $cache,
-        #[Autowire('%kernel.debug%')]
-        private bool $isDebug,
-        #[Autowire(service: 'twig.command.debug')]
-        private DebugCommand $twigDebugCommand,
-    ) {}
-
-    public function findAll(): array
-    {
-        /*
-        $output = new BufferedOutput();
-        $this->twigDebugCommand->run(new ArrayInput([]), $output);
-        dd($output);
-        */
-
-        return $this->cache->get('mixes_data', function(CacheItemInterface $cacheItem) {
-            $cacheItem->expiresAfter($this->isDebug ? 5 : 60);
-            $response = $this->githubContentClient->request('GET', '/SymfonyCasts/vinyl-mixes/main/mixes.json');
-
-            return $response->toArray();
-        });
-    }
-}
