diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index d8cb9a1..c2d012a 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -42,7 +42,7 @@ class VinylController extends AbstractController
     {
         $genre = $slug ? u(str_replace('-', ' ', $slug))->title(true) : null;
 
-        $mixes = $mixRepository->findAll();
+        $mixes = $mixRepository->findBy([], ['votes' => 'DESC']);
 
         return $this->render('vinyl/browse.html.twig', [
             'genre' => $genre,
