diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index 3b1e9b8..0ac73fb 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -2,9 +2,8 @@
 
 namespace App\Controller;
 
-use App\Entity\VinylMix;
+use App\Repository\VinylMixRepository;
 use App\Service\MixRepository;
-use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
@@ -37,12 +36,10 @@ class VinylController extends AbstractController
     }
 
     #[Route('/browse/{slug}', name: 'app_browse')]
-    public function browse(EntityManagerInterface $entityManager, string $slug = null): Response
+    public function browse(VinylMixRepository $mixRepository, string $slug = null): Response
     {
         $genre = $slug ? u(str_replace('-', ' ', $slug))->title(true) : null;
 
-        $mixRepository = $entityManager->getRepository(VinylMix::class);
-        dd($mixRepository);
         $mixes = $mixRepository->findAll();
 
         return $this->render('vinyl/browse.html.twig', [
