diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index 1424757..c326354 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -25,9 +25,10 @@ class VinylController extends AbstractController
     public function homepage(
         #[MapQueryParameter] string $query = '',
         #[MapQueryParameter] int $page = 1,
+        #[MapQueryParameter(options: ['min_range' => 1, 'max_range' => 10])] int $limit = 10,
     ): Response
     {
-        dump($query, $page);
+        dump($query, $page, $limit);
         $tracks = [
             ['song' => 'Gangsta\'s Paradise', 'artist' => 'Coolio'],
             ['song' => 'Waterfalls', 'artist' => 'TLC'],
