diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index 274914c..709a2fa 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -9,6 +9,7 @@ use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\DependencyInjection\Attribute\Autowire;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Attribute\MapQueryParameter;
 use Symfony\Component\Routing\Attribute\Route;
 use function Symfony\Component\String\u;
 
@@ -21,8 +22,11 @@ class VinylController extends AbstractController
     {}
 
     #[Route('/', name: 'app_homepage')]
-    public function homepage(): Response
+    public function homepage(
+        #[MapQueryParameter] string $query = '',
+    ): Response
     {
+        dump($query);
         $tracks = [
             ['song' => 'Gangsta\'s Paradise', 'artist' => 'Coolio'],
             ['song' => 'Waterfalls', 'artist' => 'TLC'],
