diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index f66dc1f..09553fe 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -4,6 +4,7 @@ namespace App\Controller;
 
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use function Symfony\Component\String\u;
 
 class VinylController
 {
@@ -16,7 +17,7 @@ class VinylController
     #[Route('/browse/{slug}')]
     public function browse(string $slug): Response
     {
-        $title = str_replace('-', ' ', $slug);
+        $title = u(str_replace('-', ' ', $slug))->title(true);
 
         return new Response('Genre: '.$title);
 
