diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index 07c5607..f66dc1f 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -16,7 +16,9 @@ class VinylController
     #[Route('/browse/{slug}')]
     public function browse(string $slug): Response
     {
-        return new Response('Genre: '.$slug);
+        $title = str_replace('-', ' ', $slug);
+
+        return new Response('Genre: '.$title);
 
         //return new Response('Breakup vinyl? Angsty 90s rock? Browse the collection!');
     }
