diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index a8222db..d693f07 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use App\Entity\VinylMix;
 use App\Service\MixRepository;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
@@ -50,6 +51,13 @@ class VinylController extends AbstractController
     #[Route('/mix/new')]
     public function newMix(): Response
     {
-        dd('new mix');
+        $mix = new VinylMix();
+        $mix->setTitle('Do you Remember... Phil Collins?!');
+        $mix->setDescription('A pure mix of drummers turned singers!');
+        $mix->setGenre('pop');
+        $mix->setTrackCount(rand(5, 20));
+        $mix->setVotes(rand(-50, 50));
+
+        dd($mix);
     }
 }
