diff --git a/src/Controller/MixController.php b/src/Controller/MixController.php
new file mode 100644
index 0000000..7b8db1c
--- /dev/null
+++ b/src/Controller/MixController.php
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class MixController extends AbstractController
+{
+    #[Route('/mix/new')]
+    public function new(): Response
+    {
+        dd('new mix');
+    }
+}
