diff --git a/src/Controller/MixController.php b/src/Controller/MixController.php
index 7b8db1c..8313c95 100644
--- a/src/Controller/MixController.php
+++ b/src/Controller/MixController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use App\Entity\VinylMix;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
@@ -11,6 +12,13 @@ class MixController extends AbstractController
     #[Route('/mix/new')]
     public function new(): Response
     {
-        dd('new mix');
+        $mix = new VinylMix();
+        $mix->setTitle('Do you Remember... Phil Collins?!');
+        $mix->setDescription('A pure mix of drummers turned singers!');
+        $mix->setGenre('pop');
+        $mix->setTrackCount(rand(5, 20));
+        $mix->setVotes(rand(-50, 50));
+
+        dd($mix);
     }
 }
