diff --git a/src/Controller/MixController.php b/src/Controller/MixController.php
index 8313c95..9a9acfe 100644
--- a/src/Controller/MixController.php
+++ b/src/Controller/MixController.php
@@ -3,6 +3,7 @@
 namespace App\Controller;
 
 use App\Entity\VinylMix;
+use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
@@ -10,7 +11,7 @@ use Symfony\Component\Routing\Annotation\Route;
 class MixController extends AbstractController
 {
     #[Route('/mix/new')]
-    public function new(): Response
+    public function new(EntityManagerInterface $entityManager): Response
     {
         $mix = new VinylMix();
         $mix->setTitle('Do you Remember... Phil Collins?!');
@@ -19,6 +20,13 @@ class MixController extends AbstractController
         $mix->setTrackCount(rand(5, 20));
         $mix->setVotes(rand(-50, 50));
 
-        dd($mix);
+        $entityManager->persist($mix);
+        $entityManager->flush();
+
+        return new Response(sprintf(
+            'Mix %d is %d tracks of pure 80\'s heaven',
+            $mix->getId(),
+            $mix->getTrackCount()
+        ));
     }
 }
