diff --git a/src/Controller/SongController.php b/src/Controller/SongController.php
index 6c1b65d..4479db6 100644
--- a/src/Controller/SongController.php
+++ b/src/Controller/SongController.php
@@ -20,7 +20,9 @@ class SongController extends AbstractController
             'url' => 'https://symfonycasts.s3.amazonaws.com/sample.mp3',
         ];
 
-        $logger->info('Returning API response for song '.$id);
+        $logger->info('Returning API response for song {song}', [
+            'song' => $id,
+        ]);
 
         return $this->json($song);
     }
