diff --git a/src/Controller/VinylController.php b/src/Controller/VinylController.php
index 4c667c4..166fe58 100644
--- a/src/Controller/VinylController.php
+++ b/src/Controller/VinylController.php
@@ -5,13 +5,12 @@ namespace App\Controller;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
-use Twig\Environment;
 use function Symfony\Component\String\u;
 
 class VinylController extends AbstractController
 {
     #[Route('/', name: 'app_homepage')]
-    public function homepage(Environment $twig): Response
+    public function homepage(): Response
     {
         $tracks = [
             ['song' => 'Gangsta\'s Paradise', 'artist' => 'Coolio'],
@@ -22,12 +21,10 @@ class VinylController extends AbstractController
             ['song' => 'Fantasy', 'artist' => 'Mariah Carey'],
         ];
 
-        $html = $twig->render('vinyl/homepage.html.twig', [
+        return $this->render('vinyl/homepage.html.twig', [
             'title' => 'PB & Jams',
             'tracks' => $tracks,
         ]);
-
-        return new Response($html);
     }
 
     #[Route('/browse/{slug}', name: 'app_browse')]
