diff --git a/src/Controller/SongController.php b/src/Controller/SongController.php
index 59bfc14..6c1b65d 100644
--- a/src/Controller/SongController.php
+++ b/src/Controller/SongController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Response;
@@ -10,7 +11,7 @@ use Symfony\Component\Routing\Annotation\Route;
 class SongController extends AbstractController
 {
     #[Route('/api/songs/{id<\d+>}', methods: ['GET'])]
-    public function getSong(int $id): Response
+    public function getSong(int $id, LoggerInterface $logger): Response
     {
         // TODO query the database
         $song = [
@@ -19,6 +20,8 @@ class SongController extends AbstractController
             'url' => 'https://symfonycasts.s3.amazonaws.com/sample.mp3',
         ];
 
+        $logger->info('Returning API response for song '.$id);
+
         return $this->json($song);
     }
 }
