diff --git a/src/Controller/MixController.php b/src/Controller/MixController.php
index 9e7ebbb..241895f 100644
--- a/src/Controller/MixController.php
+++ b/src/Controller/MixController.php
@@ -30,4 +30,10 @@ class MixController extends AbstractController
             $mix->getTrackCount()
         ));
     }
+
+    #[Route('/mix/{id}')]
+    public function show($id): Response
+    {
+        dd($id);
+    }
 }
