diff --git a/src/Controller/MixController.php b/src/Controller/MixController.php
index 241895f..d5320dc 100644
--- a/src/Controller/MixController.php
+++ b/src/Controller/MixController.php
@@ -3,6 +3,7 @@
 namespace App\Controller;
 
 use App\Entity\VinylMix;
+use App\Repository\VinylMixRepository;
 use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
@@ -32,8 +33,9 @@ class MixController extends AbstractController
     }
 
     #[Route('/mix/{id}')]
-    public function show($id): Response
+    public function show($id, VinylMixRepository $mixRepository): Response
     {
-        dd($id);
+        $mix = $mixRepository->find($id);
+        dd($mix);
     }
 }
