diff --git a/src/Controller/MixController.php b/src/Controller/MixController.php
index 54c01d0..c98a2b2 100644
--- a/src/Controller/MixController.php
+++ b/src/Controller/MixController.php
@@ -32,7 +32,7 @@ class MixController extends AbstractController
         ));
     }
 
-    #[Route('/mix/{id}')]
+    #[Route('/mix/{id}', name: 'app_mix_show')]
     public function show($id, VinylMixRepository $mixRepository): Response
     {
         $mix = $mixRepository->find($id);
diff --git a/templates/vinyl/browse.html.twig b/templates/vinyl/browse.html.twig
index 2d21c54..c977861 100644
--- a/templates/vinyl/browse.html.twig
+++ b/templates/vinyl/browse.html.twig
@@ -28,7 +28,9 @@
         <div class="row">
             {% for mix in mixes %}
             <div class="col col-md-4">
-                <div class="mixed-vinyl-container p-3 text-center">
+                <a href="{{ path('app_mix_show', {
+                    id: mix.id
+                }) }}" class="mixed-vinyl-container p-3 text-center">
                     <img src="{{ mix.getImageUrl(300) }}" alt="Mix album cover">
                     <p class="mt-2"><strong>{{ mix.title }}</strong></p>
                     <span>{{ mix.trackCount }} Tracks</span>
@@ -38,7 +40,7 @@
                     <span>{{ mix.createdAt|ago }}</span>
                     <br>
                     {{ mix.votesString }} votes
-                </div>
+                </a>
             </div>
             {% endfor %}
         </div>
