diff --git a/src/Controller/MixController.php b/src/Controller/MixController.php
index c98a2b2..e9b0334 100644
--- a/src/Controller/MixController.php
+++ b/src/Controller/MixController.php
@@ -37,6 +37,10 @@ class MixController extends AbstractController
     {
         $mix = $mixRepository->find($id);
 
+        if (!$mix) {
+            throw $this->createNotFoundException('Mix not found');
+        }
+
         return $this->render('mix/show.html.twig', [
             'mix' => $mix,
         ]);
