diff --git a/templates/vinyl/homepage.html.twig b/templates/vinyl/homepage.html.twig
index 75e3c52..55e2357 100644
--- a/templates/vinyl/homepage.html.twig
+++ b/templates/vinyl/homepage.html.twig
@@ -33,7 +33,7 @@
             </svg>
         </div>
         <div class="col-12 col-md-8 ps-5">
-            <div data-controller="hello"></div>
+            <div {{ stimulus_controller('hello') }}></div>
             <h2 class="mb-4">10 songs (30 minutes of 60 still available)</h2>
             {% for track in tracks %}
             <div class="song-list">
