diff --git a/templates/vinyl/homepage.html.twig b/templates/vinyl/homepage.html.twig
index 144d942..9a073e7 100644
--- a/templates/vinyl/homepage.html.twig
+++ b/templates/vinyl/homepage.html.twig
@@ -8,7 +8,7 @@
     <ul>
         {% for track in tracks %}
             <li>
-                {{ track }}
+                {{ track.song }} - {{ track.artist }}
             </li>
         {% endfor %}
     </ul>
