diff --git a/templates/vinyl/homepage.html.twig b/templates/vinyl/homepage.html.twig
index 4143b61..3d6c077 100644
--- a/templates/vinyl/homepage.html.twig
+++ b/templates/vinyl/homepage.html.twig
@@ -8,7 +8,7 @@
     <ul>
         {% for track in tracks %}
             <li>
-                {{ track.song }} - {{ track.artist|upper }}
+                {{ track.song }} - {{ track.artist|upper|lower }}
             </li>
         {% endfor %}
     </ul>
