diff --git a/config/secrets/prod/prod.GITHUB_TOKEN.28bd2f.php b/config/secrets/prod/prod.GITHUB_TOKEN.28bd2f.php
new file mode 100644
index 0000000..9b238b7
--- /dev/null
+++ b/config/secrets/prod/prod.GITHUB_TOKEN.28bd2f.php
@@ -0,0 +1,3 @@
+<?php // prod.GITHUB_TOKEN.28bd2f on Mon, 09 May 2022 18:31:56 +0000
+
+return "\xE6ji\x2A\xEB\xD8\xAD\xA0\xD75\xB4\xF1\xE4\xB2\xE3\xA8\x28\xC1\xCD\x21\xB5\xB0\xFB\x87\x08\xF7\xA1\x05\x99\x9E\xD5h\x2B\x09\xB0\x2B\xAB\xC1\x27u\x14hR\xCD\xE4\x5D\x95\xD7\x84\xBA\x1A\xDA\xF88\x0B\x24\x85\xFC\xF2\xB3d\x99TW\xFE\xAE\x196\x0EM\x28\x94\x9Dx\xAB\x97e6\xF5\xAF\xF7z\xD5\x994y\x40\xCF";
diff --git a/config/secrets/prod/prod.encrypt.public.php b/config/secrets/prod/prod.encrypt.public.php
new file mode 100644
index 0000000..b7d7105
--- /dev/null
+++ b/config/secrets/prod/prod.encrypt.public.php
@@ -0,0 +1,3 @@
+<?php // prod.encrypt.public on Mon, 09 May 2022 18:31:55 +0000
+
+return "C\xD1hg\x7D\xB5\xCF\xA2~\x0D\x0FD\x99\xDDf~\x02\x92\xCB~\xF8\x27\x20\x8B\xCF~\xA5\x1F\xA4o\xBA\x26";
diff --git a/config/secrets/prod/prod.list.php b/config/secrets/prod/prod.list.php
new file mode 100644
index 0000000..d2f5bed
--- /dev/null
+++ b/config/secrets/prod/prod.list.php
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'GITHUB_TOKEN' => null,
+];
