diff --git a/src/Service/MixRepository.php b/src/Service/MixRepository.php
index 87ee9d6..98b1dc0 100644
--- a/src/Service/MixRepository.php
+++ b/src/Service/MixRepository.php
@@ -29,6 +29,7 @@ class MixRepository
         $this->twigDebugCommand->run(new ArrayInput([]), $output);
         dd($output);
         */
+        dd($this->githubContentClient);
 
         return $this->cache->get('mixes_data', function(CacheItemInterface $cacheItem) {
             $cacheItem->expiresAfter($this->isDebug ? 5 : 60);
