diff --git a/src/Controller/MixController.php b/src/Controller/MixController.php
index a93005b..d5ff784 100644
--- a/src/Controller/MixController.php
+++ b/src/Controller/MixController.php
@@ -44,6 +44,6 @@ class MixController extends AbstractController
     #[Route('/mix/{id}/vote', name: 'app_mix_vote', methods: ['POST'])]
     public function vote(VinylMix $mix, Request $request): Response
     {
-        dd($request->request->get('direction'));
+        $direction = $request->request->get('direction', 'up');
     }
 }
